config.yml

```diff
@@ -111,6 +112,7 @@ DoPreview: true
 DOFSThreshold: 0
 
 ## Resource allocation settings for slurm jobs
+PriorMemory: 50000
```

Comment:
A few things: 
1) Perhaps the default should be set to 32000 to be consistent with `SimulationMemory`. Do you think we should have `PriorEmis` use the configured `SimulationMemory` or is there benefit to keeping these variables separate? 

2) Should there also be `CombineJacobianRuns` somewhere in config.yml or is it assumed false if not present? I dont actually see this variable referenced anywhere.. is this used?

3) Can you add new config variables in sanitize_input_yaml.py? This would also be a good place to enforce that if combining jacobian run, UseTotalPriorEmis is true.

Reply:
> 1. Perhaps the default should be set to 32000 to be consistent with `SimulationMemory`. Do you think we should have `PriorEmis` use the configured `SimulationMemory` or is there benefit to keeping these variables separate?

Yes, I think it makes sense for them to be consistent. I believe I had separated them under the impression that the HEMCO standalone would require drastically different memory allocation but they're close enough that they can use the same value.

> 2. Should there also be `CombineJacobianRuns` somewhere in config.yml or is it assumed false if not present? I dont actually see this variable referenced anywhere.. is this used?

No, the `CombineJacobianRuns` logical was removed during some of the cleanup. It's now determined simply from `NumJacobianRuns` -- if that value is >0 then the jacobian runs will be combined into the specified number. I've updated the description of this pull request to avoid confusion.

> 3. Can you add new config variables in sanitize_input_yaml.py? This would also be a good place to enforce that if combining jacobian run, UseTotalPriorEmis is true.

Both `CombineJacobianRuns` and `UseTotalPriorEmis` were removed in later commits of this PR during cleanup. The IMI will always use total prior emissions (unless a user goes in and manually changes that in `template_run/HEMCO_Config.rc` before setting up the jacobian runs). This was changed because it simplifies applying the emissions perturbations in HEMCO_Config.rc.

With that said, I have updates `sanitize_input_yaml.py` to include new fields added to the config file in this PR.